main.go

```diff
@@ -28,6 +28,11 @@ var (
 	ErrInvalidTimeouts = errors.New("appendInterval must be shorter than minimum election window")
 )
 
+const (
```

Comment:
This variable does not get correctly overwritten if the variable is a const. Switch this to a var and it works as expected

Reply:
Oh, snap! I think I missed this while refactoring it. Sorry.